pom.xml

```diff
@@ -164,7 +173,7 @@
     <dependency>
       <groupId>net.snowflake</groupId>
       <artifactId>snowflake-common</artifactId>
-      <version>4.0.1</version>
+      <version>5.0.1</version>
```

Comment:
https://github.com/snowflakedb/snowflake/pull/12642 for version 5.0.1. Need to wait for this release.